wmn-data.json

```diff
@@ -1870,7 +1870,7 @@
        },
        {
         "name" : "Fotolog Archived Profile",
-        "uri_check" : "http://archive.org/wayback/available?url=fotolog.com/{account}",
+        "uri_check" : "http://archive.org/wayback/available?url=http://www.fotolog.com/{account}",

```

Comment:
please make this `https`

Reply:
I believe both `http` could be changed to `https` like `https://archive.org/wayback/available?url=https://www.fotolog.com/x_zudex_x`